updated-env-model (#1549)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/schema/schema-metadata-model.mdx

}
```

#### Column-level information

> The example query below could be useful if we wanted to understand more information about the columns of a given model.  Note that this will only work if the job generated documentation (for example, with command 'dbt docs generate').

Reviewer: ```suggestion
You can use the following example query to understand more about the columns of a given model.  Note that this will only work if the job has generated documentation. For example it will work with the command `dbt docs generate`.
```
